scripts/es-integration-test.sh

```diff
@@ -96,14 +96,32 @@ teardown_storage() {
   docker compose -f "${compose_file}" down
 }
 
+build_local_img(){
+    local LOCAL_FLAG=''
+    local platforms="linux/amd64"
+    # Loop through each platform (separated by commas)
+    for platform in $(echo "$platforms" | tr ',' ' '); do
+      # Extract the architecture from the platform string
+      arch=${platform##*/}  # Remove everything before the last slash
+      make "build-binaries-$arch"
+    done
```

Comment:
```suggestion
    make "build-binaries-linux"
```
Is that what you are trying to do?

Reply:
yes but i let i be there since i wasn't sure if we have to test only on linux/amd64  i will make the changes